Flabbergasted-Flamingos/Bangazon-admin-app
# Seeder

# Description
This merge adds faker functionality for this app.  Inside a new directory: admin_app/management/commands there is a file called seeder.py, containing the code for the seeder command that we will use to seed the database with customized data for testing.  

## Number of Fixes
1

## Related Ticket(s)
None

## Problem to Solve
See description

## Proposed Changes
1. Adds seeder to installed apps, 
2. adds management/commands path to give manage.py the command "seeder"
3. adds seeder.py file to customize the data to be seeded. 

## Expected Behavior
after remigrating, dev should be able to use `python manage.py seeder` and add data automagically to the database.  

## Steps to Test Solution
Make sure you are in your virtual environment. 

```
git fetch --all
git checkout seeder
pip install django-seed
rm -rf admin_app/migrations
rm db.sqlite3
python manage.py makemigrations admin_app
python manage.py migrate
python manage.py seeder

## Testing
Open db in sqlite3 or in DB Browsers and check to see if the data seeded properly.
